.gitignore

```diff
@@ -32,4 +32,12 @@ tests
 .gradle
 **/build/
 !src/**/build/
-/gradle
```

Comment:
this folder contains important jars for the gradle wrapper and should not be ignored